django/core/management/commands/shell.py

```diff
@@ -47,18 +48,20 @@ def add_arguments(self, parser):
     def ipython(self, options):
         from IPython import start_ipython
 
-        start_ipython(argv=[])
+        imported_objects = self.get_namespace(options)
+        start_ipython(argv=[], user_ns=imported_objects)
```

Comment:
No need for a temporary variable, and the same for the `bpython()` method.

```suggestion
        start_ipython(argv=[], user_ns=self.get_namespace(options))
```